rakelib/default_plugins.rb

```diff
@@ -119,6 +119,8 @@ module RakeLib
       logstash-input-stdin
       logstash-input-tcp
       logstash-output-stdout
+      logstash-output-null
+      logstash-codec-rubydebug
```

Comment:
@purbon I dont believe to be the source of the problem, since the custom pipeline in your test is not using the rubydebug at all? So I am not sure where its used?